webapp/src/main/java/edu/agh/yait/HelloController.java

```diff
@@ -1,19 +1,65 @@
 package edu.agh.yait;
 
+import edu.agh.yait.userData.UserData;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.util.Arrays;
+import java.util.List;
+
 /**
  * Created by mf57 on 05.04.2017.
  */
+
+//TODO Endpoint for debugging, to be removed
 @RestController
 @RequestMapping("/api")
 public class HelloController {
-
-    @RequestMapping(value = "/hello", method = RequestMethod.GET)
-    public String hello() {
-        return "Hello World";
-    }
+//
```

Comment:
to narazie niech zostanie zakomentowane, bo pewnie jeszcze trzeba będzie jakies przerobki w tym ldapie zrobic przy integracji, a tak sie to najszybciej testuje